<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1166</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>950</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="autoFillBackground">
       <bool>true</bool>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Triangular</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="tabsClosable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="icon">
        <iconset>
         <normaloff>:/test/ui/resources/img/process.png</normaloff>:/test/ui/resources/img/process.png</iconset>
       </attribute>
       <attribute name="title">
        <string/>
       </attribute>
       <attribute name="toolTip">
        <string>Home</string>
       </attribute>
       <attribute name="whatsThis">
        <string>Home Table</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTableWidget" name="mainTable">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>901</width>
            <height>501</height>
           </size>
          </property>
          <property name="contextMenuPolicy">
           <enum>Qt::CustomContextMenu</enum>
          </property>
          <property name="sizeAdjustPolicy">
           <enum>QAbstractScrollArea::AdjustToContents</enum>
          </property>
          <property name="sortingEnabled">
           <bool>true</bool>
          </property>
          <attribute name="horizontalHeaderVisible">
           <bool>false</bool>
          </attribute>
          <attribute name="horizontalHeaderCascadingSectionResizes">
           <bool>true</bool>
          </attribute>
          <attribute name="horizontalHeaderDefaultSectionSize">
           <number>75</number>
          </attribute>
          <attribute name="horizontalHeaderStretchLastSection">
           <bool>true</bool>
          </attribute>
          <attribute name="verticalHeaderStretchLastSection">
           <bool>false</bool>
          </attribute>
          <column>
           <property name="text">
            <string>URL</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>PWD</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>TYPE</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ENCODING</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>CATEGORY</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>SQLCONF</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>TAG</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>IPGEO</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>STATUS</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>CREATEAT</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>UPDATEAT</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar_file">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_import"/>
   <addaction name="action_export"/>
   <addaction name="action_quit"/>
  </widget>
  <widget class="QToolBar" name="toolBar_action">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_file"/>
   <addaction name="action_cmd"/>
   <addaction name="action_data"/>
  </widget>
  <widget class="QToolBar" name="toolBar_about">
   <property name="windowTitle">
    <string>toolBar_3</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_about"/>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1166</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_file">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="action_import"/>
    <addaction name="action_export"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <widget class="QMenu" name="menu_about">
    <property name="title">
     <string>关于</string>
    </property>
    <addaction name="action_about"/>
   </widget>
   <addaction name="menu_file"/>
   <addaction name="menu_about"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action_add"/>
   <addaction name="action_edit"/>
   <addaction name="action_del"/>
  </widget>
  <action name="action_import">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/test/ui/images/import.png</normaloff>:/test/ui/images/import.png</iconset>
   </property>
   <property name="text">
    <string>导入</string>
   </property>
  </action>
  <action name="action_export">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/test/ui/images/export.png</normaloff>:/test/ui/images/export.png</iconset>
   </property>
   <property name="text">
    <string>导出</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/test/ui/images/power_exit.png</normaloff>:/test/ui/images/power_exit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="action_file">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/test/ui/images/folder.png</normaloff>:/test/ui/images/folder.png</iconset>
   </property>
   <property name="text">
    <string>文件管理</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+F</string>
   </property>
  </action>
  <action name="action_cmd">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/test/ui/images/terminal.png</normaloff>:/test/ui/images/terminal.png</iconset>
   </property>
   <property name="text">
    <string>终端模拟</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+C</string>
   </property>
  </action>
  <action name="action_data">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/test/ui/images/process.png</normaloff>:/test/ui/images/process.png</iconset>
   </property>
   <property name="text">
    <string>数据库管理</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+D</string>
   </property>
  </action>
  <action name="action_about">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/test/ui/images/restart1.png</normaloff>:/test/ui/images/restart1.png</iconset>
   </property>
   <property name="text">
    <string> About</string>
   </property>
   <property name="toolTip">
    <string>关于</string>
   </property>
  </action>
  <action name="action_add">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/test/ui/images/add.png</normaloff>:/test/ui/images/add.png</iconset>
   </property>
   <property name="text">
    <string>添加</string>
   </property>
   <property name="toolTip">
    <string>添加Shell</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
  <action name="action_edit">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/test/ui/images/edit.png</normaloff>:/test/ui/images/edit.png</iconset>
   </property>
   <property name="text">
    <string>编辑</string>
   </property>
   <property name="toolTip">
    <string>编辑Shell</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+E</string>
   </property>
  </action>
  <action name="action_del">
   <property name="icon">
    <iconset resource="../../resource.qrc">
     <normaloff>:/test/ui/images/delete.png</normaloff>:/test/ui/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
   <property name="toolTip">
    <string>删除Shell</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+R</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../../resource.qrc"/>
 </resources>
 <connections/>
</ui>
